### ISSUE TITLE
Cut screen when loading room/home screens

### ISSUE THREAD

**Issue by ghost** [07151129/sh3proxy#14]
With the latest sh3proxy version I find that the screen is like cut when loading rooms or in konami logo, I have american 5cd's version but I don't know if is for it..

Screens showing the error:

http://imgur.com/a/3eV4Y

**Comment by CHNSK** [comment]
Check your rendering resolution. It's probably same as your display resolution. Try to use game's own values from options and don't force from .ini.

**Comment by 07151129** [comment]
That's interesting; overriding internal resolution didn't cause tearing when testing. What is your screen (SizeX/SizeY) resolution, are you using the resolution override (MatchRes), and what is the rendering resolution set in game options?

**Comment by ghost** [comment]
I use 1360x768, and yes matchres = 1

Edit: I put matchres = 0 and select res in options game and work like Markeron say

P.S: other thing that I don't know if other "issue" is, the cheats (godmode and unlimitedammo = 1, don't work for me or there is any steps to work)

**Comment by argvx** [comment]
It seems that crc32 of your sh3.exe is differs from 372ae792, I don't have any of this issues at all.
We probably need to switch from static addresses to byte array pattern search..
I already wrote a simplified version of Hooking.Pattern from WS Pack in C, but needs more testing and it missing rva offsets...

**Comment by 07151129** [comment]
> We probably need to switch from static addresses to byte array pattern search..

I'd rather have a table of addresses per game version; byte matching is probably an overkill.

**Comment by argvx** [comment]
> I'd rather have a table of addresses per game version; byte matching is probably an overkill.

Not at all, if done right it can help getting work with many versions of modified exe without any problems, and that is a good reason to make it.
I think, that have a table of addresses per game version and then fixing many problems with each, this is a real overkill.

**Comment by 07151129** [comment]
> I think, that have a table of addresses per game version and then fixing many problems with each, this is a real overkill.

If we're lucky, we can get away with offsetting addresses…
